showcase/tslint.json

```diff
@@ -10,6 +10,7 @@
     "component-selector": false,
     "no-unused-variable": false, // FIXME: disabled due to false positives in imports. Re-enable once these are solved in a newer version of TSLint
     "jsdoc-format": [true, "check-multiline-start"],
-    "no-redundant-jsdoc": true
+    "no-redundant-jsdoc": true,
+    "selector-pseudo-element-no-unknown": [true, { "ignorePseudoElements": ["ng-deep"] }]
```

Comment:
Right, this rule is needed but it should be added to the `.stylelintrc` instead because it is part of Stylelint and not TSLint 😉